examples/tests/error/repeat_construct_id2.w

```diff
@@ -1,6 +1,6 @@
 bring cloud;
 
-let make_name = (): str => {
+let make_name = () -> str {
```

Comment:
I am wondering if it makes sense to only use the `->` arrow syntax for function *declarations* but keep the *definitions* with `:`:

```js
struct Foo {
  predicate: (str) -> bool;
}

But then:

Foo {
  predicate: (x: str): bool => true;
}
```

This means that if I want to declare a function that returns a function it will look like this:

```js
let returnsPredicate = (): (str) -> bool => {
  return (x: str) => true;
};
```

Reply:
Interesting alternative 🤔 . You've also something else, which is whether we will have a shorter syntax for closures that just return a single expression. Ignoring the type annotation stuff, today in Wing you have to write

```js
let returnsPredicate = (): (str): bool => {
  return (x: str): bool => { return true; };
};
```

But it could be nice to support something like
```js
let returnsPredicate = (): (str): bool => {
  return (x: str): bool => true;
};

Reply:
yes, `true` instead of `{ return true; }` is definitely something we would want to introcuce.